src/Transports/MassTransit.ActiveMqTransport/Topology/Topologies/ActiveMqConsumeTopology.cs

```diff
@@ -71,17 +74,20 @@ public void Bind(string topicName, Action<ITopicBindingConfigurator> configure =
         public override string CreateTemporaryQueueName(string tag)
         {
             var result = base.CreateTemporaryQueueName(tag);
-            return new string(result.Where(c => c != '.').ToArray());
+            var tempName = new string(result.Where(c => c != '.').ToArray());
+            return tempName;
         }
 
         public override IEnumerable<ValidationResult> Validate()
         {
             return base.Validate().Concat(_specifications.SelectMany(x => x.Validate()));
         }
 
+        public IActiveMqTopologyConfiguration Topology => _topology;

```

Comment:
This seems wrong, referencing a container object for the other topologies.

Reply:
new commits have different implementation